### DISCUSSION TITLE
Data analysis

### DISCUSSION THREAD

**Discussion by veslugleynat** [vdemichev/DiaNN#784] [General]
Hi, 
Firstly I apologise as I am very new to mass spec proteomics and DIA. I want to do quantitative analysis on my DIA-NN data. I have heard from others that they use MSqRob R package with MaxQuant data. Can I implement this to DIA-NN data or is there any other package/workflow that I can follow? 
I am very lost so I appreciate any guidance. Thank you in advance!

**Comment by vdemichev** [comment]
The R packages are to be used downstream of DIA-NN, i.e. you just input DIA-NN analysis into them. I think importing in MSqRob is pretty easy.

Best,
Vadim

**Reply by vdemichev** [reply]
You can also just analyse the matrices produces by DIA-NN in Excel.
